stl/inc/xutility

```diff
@@ -3695,39 +3695,82 @@ struct _Iterator_base12;
 struct _Default_sentinel {}; // empty struct to serve as the end of a range
 
 // CLASS TEMPLATE move_iterator
+#ifdef __cpp_lib_concepts
+template <semiregular>

```

Comment:
Should we include a parameter name in this forward declaration?

Reply:
Unreferenced names are at best documentation; I see no need to document this forward declaration of a standardized library entity. Folks who care can "Peek Definition" or look at cppreference.

Reply:
Ok.